src/apprt/gtk/winproto/wayland.zig

```diff
@@ -356,53 +356,66 @@ pub const Window = struct {
 
     fn syncQuickTerminal(self: *Window) !void {
         const window = self.apprt_window.window.as(gtk.Window);
-        const position = self.apprt_window.config.quick_terminal_position;
+        const config = self.apprt_window.config;
```

Comment:
Not sure if you want a `&` here to make it a pointer or if you actually want to copy the full config structure.